seq2science/rules/bam_cleaning.smk

```diff
@@ -241,6 +262,21 @@ rule samtools_index:
         samtools index {params} {input} {output}
         """
 
+rule sam2bam:
```

Comment:
nothing wrong with this rule, but any error with bam files in the future will point to this rule!

Reply:
Yes, maybe I will put an if statement around it